crates/ruff_python_ast/src/node.rs

```diff
@@ -4430,9 +4430,7 @@ impl AstNode for ast::FString {
     where
         V: PreorderVisitor<'a> + ?Sized,
     {
-        let ast::FString { elements, range: _ } = self;
-
-        for fstring_element in elements {
+        for fstring_element in &self.elements {
```

Comment:
Thanks, that makes sense.

Reply:
I don't think the field should be private for consistency. All our AST fields are public; I would rather change this holistically or it becomes confusing when using the AST node because you never know if destructuring is possible or not.

Reply:
Hmm. Maybe I could use a newtype, like I did with the bitflag for the tokens in https://github.com/astral-sh/ruff/pull/10256, so that the `flag` field is public but its internals are private.